api/scripts/wazuh-apid.py

```diff
@@ -75,6 +75,10 @@ def print_version():
 
 def main(cors, port, host, ssl_context):
     app = connexion.App(__name__, specification_dir=f'{common.ossec_path}/api/api/spec/')
+    for h in logger.handlers:
```

Comment:
These handlers are going to be added in #2739

Reply:
Fixed!